### DISCUSSION TITLE
Upgrade to Enterprise account

### DISCUSSION THREAD

**Discussion by ozonni** [community/community#39380] [Enterprise]
I've started 30 days Enterprise plan trial and would like to convert my account to Enterprise account. Since i don't have self-service button, i've contacted support.

But support said the following:
```
It is impossible to switch your account to an Enterprise Account by paying via credit cared on a monthly plan. You can only do this by paying via invoice on a yearly basis.
```

So i can't stay on monthly plan and pay by CC although i can't find anywhere in documentation that this is a case and i have to pay yearly.

Did anyone have similar issues?

**Comment by laughedelic** [comment]
Yes, I had the same issue. You can't pay for an Enterprise account via credit card. If you want an Enterprise account, you will have to switch to paying by invoice.

Relevant docs:
- https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-your-github-account/about-billing-for-your-enterprise
- https://docs.github.com/en/enterprise-cloud@latest/admin/overview/creating-an-enterprise-account

I don't see any explicit mention of not being able to pay with a credit card, but I think it's implied by this paragraph:

> If you pay by invoice, you can create an enterprise account directly through GitHub. If you do not currently pay by invoice, you can [contact our sales team](https://github.com/enterprise/contact?ref_page=/pricing&ref_cta=Contact%20Sales&ref_loc=cards) to create an enterprise account for you.

**Reply by ozonni** [reply]
paying by invoice would be fine, but can you keep monthly payments?

**Reply by laughedelic** [reply]
That I don't know, I think you need to talk to the sales team about it.
